### PR TITLE
Newsletter #99

### PR DESCRIPTION
**Pull request by p16**

### CODE REVIEW

File: source/_posts/2015-07-16-newsletter-number-99.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+layout: post
+title: "Newsletter #99"
+date: 2015-07-16 14:06
+comments: true
+categories: [ js, ux, agile, angularjs, antipatterns, css, design, devops, docker, git, github, golang, jquery, js, lisp, lodash, microservices, php, phptest codeception, reactjs, retro, symfony, sysadmin, system, terminal, testing, ui]
+author: Filippo De Santis
+---
+## It's TechThursday again!
+
+And here we are with what we read this week! Enjoy!
```
Comment (geshan) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: nothing about EID?

File: source/_posts/2015-07-16-newsletter-number-99.markdown [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+* [Inbox (8) - alessandro.cinelli@namshi.com - Namshi MailThe web is fast by default, let’s keep it fast](https://hiddedevries.nl/en/blog/2015-05-18-the-web-is-fast-by-default-lets-keep-it-fast)
+
+* [Introducing Incremental DOM](https://medium.com/google-developers/introducing-incremental-dom-e98f79ce2c5f)
+
+* [Just Don’t Hire 0x Engineers](http://zachholman.com/posts/0x-engineers/)
+
+* [MIT comes up with a "no memory" solution for Big Data | ITworld](http://www.itworld.com/article/2947839/big-data/mit-comes-up-with-a-no-memory-solution-for-big-data.html)
+
+* [MIT comes up with a "no memory" solution for Big Data](http://www.itworld.com/article/2947839/big-data/mit-comes-up-with-a-no-memory-solution-for-big-data.html)
+
```
Comment (prabeesh) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: duplicate

Comment (p16) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: thanks, I'll remove one

File: source/_posts/2015-07-16-newsletter-number-99.markdown [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+* [ShipIt by Shopify](https://github.com/Shopify/shipit-engine)
+
+* [Small Go Apps Containers](http://carlosbecker.com/posts/small-go-apps-containers/)
+
+* [Software Engineering Blogs](https://github.com/kilimchoi/engineering-blogs)
+
+* [The 10 Coolest Big Data Products Of 2015 (So Far) - Page: 1 | CRN](http://www.crn.com/slide-shows/applications-os/300077353/the-10-coolest-big-data-products-of-2015-so-far.htm)
+
+* [The 10 Coolest Big Data Products Of 2015 (So Far)](http://www.crn.com/slide-shows/applications-os/300077353/the-10-coolest-big-data-products-of-2015-so-far.htm)
+
```
Comment (prabeesh) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: again

File: source/_posts/2015-07-16-newsletter-number-99.markdown [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+* [multi-lingual decentralized p2p chat based on web RTC and Google Translate API](https://github.com/marcacohen/polychat)
+
+* [ng-model-options: Total Model Control](https://www.airpair.com/angularjs/posts/ngmodeloptions-total-model-control?utm_content=bufferf25bb)
+
+* [progrium/go-shell](https://github.com/progrium/go-shell)
+
+* [MIT comes up with a "no memory" solution for Big Data](http://www.itworld.com/article/2947839/big-data/mit-comes-up-with-a-no-memory-solution-for-big-data.html)
+
+* [The 10 Coolest Big Data Products Of 2015 (So Far)](http://www.crn.com/slide-shows/applications-os/300077353/the-10-coolest-big-data-products-of-2015-so-far.htm)
+
```
Comment (prabeesh) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: same link in the L88

Comment (p16) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: @prabeesh I've removed all the duplicates ;)

File: source/_posts/2015-07-16-newsletter-number-99.markdown [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+* [google/incremental-dom](https://github.com/google/incremental-dom)
+
+* [multi-lingual decentralized p2p chat based on web RTC and Google Translate API](https://github.com/marcacohen/polychat)
+
+* [ng-model-options: Total Model Control](https://www.airpair.com/angularjs/posts/ngmodeloptions-total-model-control?utm_content=bufferf25bb)
+
+* [progrium/go-shell](https://github.com/progrium/go-shell)
+
+* [MIT comes up with a "no memory" solution for Big Data](http://www.itworld.com/article/2947839/big-data/mit-comes-up-with-a-no-memory-solution-for-big-data.html)
+
```
Comment (prabeesh) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: same link in the L64

File: source/_posts/2015-07-16-newsletter-number-99.markdown (hunk 4 above)
Comment (prabeesh) [inline source/_posts/2015-07-16-newsletter-number-99.markdown]: thanks

**Pull request merged**